packages/babel-generator/src/generators/types.js

```diff
@@ -142,15 +142,27 @@ export function StringLiteral(node: Object, parent: Object) {
     return;
   }
 
-  // ensure the output is ASCII-safe
-  const opts = {
-    quotes: t.isJSX(parent) ? "double" : this.format.quotes,
-    wrap: true
-  };
-  if (this.format.jsonCompatibleStrings) {
-    opts.json = true;
+  let val;
+  if (this.format.disableASCIIEncode) {
```

Comment:
Yes, it's was my first thoughts, but minimal option it's new feature for jsesc and this not released at this time :(